### PR TITLE
chore: Signup remove direct usage of theming

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Developer experience (improves developer workflows for contributing to the project)

## Description

The intent here is to remove direct use of theming from the SPA and instead use [theme aware components](https://theme-ui.com/theme-spec).

### CODE REVIEW

File: src/pages/SignUp/SignUp.tsx [hunk 1]
```diff
@@ -233,15 +217,19 @@ class SignUpPage extends React.Component<IProps, IState> {
...
                             type="checkbox"
                             component="input"
                             validate={required}
                           />
-                          <Text>
+                          <Text
+                            sx={{
+                              fontSize: 2,
+                            }}
+                          >
```
Comment (AlfonsoGhislieri) [inline src/pages/SignUp/SignUp.tsx:232]: **Nitpick** should the fontsize not be applied to the label component?

**Review by AlfonsoGhislieri — APPROVED** [review]
Looks good @thisislawatts,  I'm guessing we should start looking at implementing this across other pages and components?

File: src/pages/SignUp/SignUp.tsx (hunk 1 above)
Comment (thisislawatts) [reply src/pages/SignUp/SignUp.tsx:232]: I _think_ this affected the sizing/spacing around the Checkbox which raises some questions, I backed out of that change and opted for the more focused styling approach.

**Comment by thisislawatts** [comment]
That's right @AlfonsoGhislieri, my intent is to work through these instances. At time of writing there are **46** across the site.
```bash
grep -Rnw "./src/" -e "Remove direct usage of Theme" | wc -l
```

I see there being 2 primary options:
1. Refactor to use standard components and access theme within [`sx` prop](https://theme-ui.com/sx-prop)
2. Create new component in `./packages/components`

```
./src/common/Form/ImageInput/ImageInput.tsx
./src/common/Form/ImageInput/ImageConverter.tsx
./src/common/Form/FieldContainer.ts
./src/modules/admin/components/AdminSubheader.tsx
./src/modules/admin/pages/adminApprovals.tsx
./src/pages/Research/Content/ResearchDescription.tsx
./src/pages/Research/Content/Common/Research.form.tsx
./src/pages/Research/Content/Common/Update.form.tsx
./src/pages/Research/Content/Common/PostingGuidelines.tsx
./src/pages/Research/Content/ResearchList.tsx
./src/pages/Research/Content/ResearchListItem.tsx
./src/pages/Maps/Content/Controls/Controls.tsx
./src/pages/Settings/SettingsPage.tsx
./src/pages/Settings/UserSettings.test.tsx
./src/pages/Settings/content/formSections/MemberMapPin.section.tsx
./src/pages/Settings/content/formSections/ChangePassword.form.tsx
./src/pages/Settings/content/formSections/Collection.section.tsx
./src/pages/Settings/content/formSections/Workspace.section.tsx
./src/pages/Settings/content/formSections/WorkspaceMapPin.section.tsx
./src/pages/Settings/content/formSections/Expertise.section.tsx
./src/pages/Settings/content/formSections/UserInfos.section.tsx
./src/pages/Settings/content/formSections/ChangeEmail.form.tsx
./src/pages/Settings/content/formSections/elements.tsx
./src/pages/Password/ForgotPassword.tsx
./src/pages/SignUp/SignUp.tsx
./src/pages/SignUp/SignUpMessage.tsx
./src/pages/SignUp/ResendSignUpMessage.tsx
./src/pages/User/content/UserStats.tsx
./src/pages/User/content/SpaceProfile.tsx
./src/pages/common/Layout/Main.tsx
./src/pages/common/Header/Menu/MenuMobile/MenuMobileExternalLink.tsx
./src/pages/common/Header/Menu/MenuMobile/MenuMobilePanel.tsx
./src/pages/common/Header/Menu/Profile/ProfileButtons.tsx
./src/pages/common/Header/Menu/Logo/Logo.tsx
./src/pages/common/Header/Menu/ProfileModal/ProfileModal.tsx
./src/pages/common/Header/Menu/Notifications/NotificationsMobile.tsx
./src/pages/common/Header/Header.tsx
./src/pages/Howto/VirtualizedFlex/VirtualizedFlex.tsx
./src/pages/Howto/DownloadExternal/DownloadExternal.tsx
./src/pages/Howto/Content/Common/Howto.form.tsx
./src/pages/Howto/Content/Common/HowtoStep.form.tsx
./src/pages/Howto/Content/Common/PostingGuidelines.tsx
./src/pages/Howto/Content/Howto/HowtoDescription/HowtoDescription.tsx
./src/pages/Howto/Content/Howto/Step/Step.tsx
./src/pages/Howto/Content/Howto/Howto.tsx
./src/pages/Events/Content/EventsCreate/EventsCreate.tsx
```

**Pull request merged**